src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -148,31 +149,31 @@ Info:
-    <string name="revanced_hide_cast_button_title">Hide cast button</string>
-    <string name="revanced_hide_cast_button_summary">Hides cast button.</string>
+    <string name="revanced_hide_playlist_card_shelf_summary">Hides the playlist card shelf in the feed.</string>
+    <string name="revanced_hide_samples_shelf_title">Hide Samples shelf</string>
+    <string name="revanced_hide_samples_shelf_summary">Hides the Samples shelf in the feed.</string>
+    <string name="revanced_hide_cast_button_title">Hide Cast button</string>
+    <string name="revanced_hide_cast_button_summary">Hides the Cast button.</string>
     <string name="revanced_hide_category_bar_title">Hide category bar</string>
-    <string name="revanced_hide_category_bar_summary">Hides category bar.</string>
+    <string name="revanced_hide_category_bar_summary">Hides the category bar.</string>
     <string name="revanced_hide_floating_button_title">Hide floating button</string>
-    <string name="revanced_hide_floating_button_summary">Hides floating button in library.</string>
+    <string name="revanced_hide_floating_button_summary">Hides the floating button in the Library tab.</string>
     <string name="revanced_hide_tap_to_update_button_title">Hide \'Tap to update\' button</string>
-    <string name="revanced_hide_tap_to_update_button_summary">Hides \'Tap to update\' button.</string>
-    <string name="revanced_hide_history_button_title">Hide history button</string>
-    <string name="revanced_hide_history_button_summary">Hides history button in toolbar.</string>
-    <string name="revanced_hide_notification_button_title">Hide notification button</string>
-    <string name="revanced_hide_notification_button_summary">Hides notification button in toolbar.</string>
+    <string name="revanced_hide_tap_to_update_button_summary">Hides the \'Tap to update\' button.</string>
+    <string name="revanced_hide_history_button_title">Hide History button</string>
+    <string name="revanced_hide_history_button_summary">Hides the History button in the toolbar.</string>
+    <string name="revanced_hide_notification_button_title">Hide Notifications button</string>
+    <string name="revanced_hide_notification_button_summary">Hides the Notifications button in the toolbar.</string>
     <string name="revanced_hide_sound_search_button_title">Hide sound search button</string>
-    <string name="revanced_hide_sound_search_button_summary">Hides sound search button in search bar.</string>
+    <string name="revanced_hide_sound_search_button_summary">Hides the sound search button in the search bar.</string>
     <string name="revanced_hide_voice_search_button_title">Hide voice search button</string>
-    <string name="revanced_hide_voice_search_button_summary">Hides voice search button in search bar.</string>
+    <string name="revanced_hide_voice_search_button_summary">Hides the voice search button in the search bar.</string>
```

Comment:
I wasn't too sure what to do here. I thought the Notifications and History buttons were proper nouns, but then it feels like the sound search and voice search buttons should also be. LMK

Reply:
According to https://github.com/inotia00/revanced-patches/pull/54#discussion_r1621890872, song search, sound search, and voice search are each written in lowercase letters 🤷

Reply:
I may be too tired, but dont you have it capitalized?

> Official string from YouTube Music:
>
> ```
> <string name="accessibility_sound_search">Sound search</string>
> <string name="accessibility_voice_search">Voice search</string>
> ```

Reply:
When I compared it with other strings, it seemed like it was capitalized because it was the beginning of a sentence

```
    <string name="accessibility_undo_add_to_library">Remove from library</string>
    <string name="accessibility_undo_dislike_video">Undo dislike</string>
    <string name="accessibility_undo_like_video">Undo like</string>
    <string name="accessibility_video_player">Video player</string>
    <string name="accessibility_video_reply_button">Attach video to reply</string>
    <string name="accessibility_voice_search">Voice search</string>
```